docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/00_Prerequisites_And_Assumptions/Database_Configuration.md

```diff
@@ -28,3 +28,41 @@ GRANT ALTER, DROP, CREATE, UPDATE, INSERT, SELECT, DELETE, CREATE TEMPORARY TABL
 
 This user's credentials should now be entered into your project's configuration file, `config.xml`
 within the `adminUser` and `adminPassword` tags.
+
+Tips:
+
+* ConfigSetting host value should not terminate in a slash. E.g. "http://localhost" not "http://localhost/". To fix, run: (for http) UPDATE Config SET Value='http://localhost' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+* If you get an error after clicking "Submit" or "Save data" on a form, check that url Config setting is set for your host. (Previous iterations of LORIS recommended setting this to the empty string) Run: UPDATE Config SET Value='_$yourhost_' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
+* If your dashboard loads but no other modules load, ensure that your apache config file (in CentOS: /var/apache2/apache2.conf) is set to AllowOverride All in the section <Directory /var/www/> to enable re-write rules (based on htdocs/.htaccess)
+* ensure your smarty/templates_c directory is writable by apache
+* run composer update to ensure your dependencies (including smarty) are up-to-date
```

Comment:
These "tips", although very useful are completely out of place. They are currently in the Database configuration file under administrative account... not the correct place for either database config values, nor filesystem permissions, nor anything else in this list. 

a better place for these might be the troubleshooting file https://github.com/aces/Loris/blob/main/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Troubleshooting.md